cmd/commands/checksum.go

```diff
@@ -0,0 +1,58 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+/* Copyright Contributors to the cpackget project. */
+
+package commands
+
+import (
+	"github.com/open-cmsis-pack/cpackget/cmd/cryptography"
+	errs "github.com/open-cmsis-pack/cpackget/cmd/errors"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+var checksumCreateCmdFlags struct {
+	// hashAlgorithm is the cryptographic hash function to be used
+	hashAlgorithm string
+
+	// outputDir is the target directory where the checksum file is written to
+	outputDir string
+}
+
+func init() {
+	ChecksumCreateCmd.Flags().StringVarP(&checksumCreateCmdFlags.hashAlgorithm, "hash-algorithm", "a", "", "specifies the hash function to be used")
```

Comment:
Later in code you do
```golang
	if hashFunction == "" {
		hashFunction = Hashes[0]
	}
```

You could use `Hashes[0]` as default value for `--hash-algorithm` flag and remove that `if` later on, leaving only 
```golang
    if !isValidHash(hashFunction) {
        return errs.ErrInvalidHashFunction
    }
```

Reply:
Done!